package.json

```diff
@@ -26,5 +29,8 @@
     "tailwindcss": "3.4.13",
     "typescript": "^5"
   },
-  "packageManager": "yarn@1.22.5+sha512.e6ecca5f074c73efa6624b57aa47e93584fdb58f158967a0539f0781ba3e297cebcf3844c8fd48a7e053b579682eb5cd70d9711496cb885286692f6f2bdca6e9"
-}
+  "resolutions": {
+    "@headlessui/react": "2.1.8"
+  },
+  "packageManager": "yarn@1.22.5"
```

Comment:
For a new project we should not be using such an old version of yarn

Reply:
we have a pbi #16  to upgrade to pnpm anyways